src/js/player.js

```diff
@@ -1292,7 +1292,13 @@ class Player extends Component {
    * @method play
    */
   play() {
-    this.techCall_('play');
+    // Only calls the tech's play if we already have a src loaded
+    if (this.player_.currentSrc()) {
+      this.techCall_('play');
+    } else {
+      this.autoplay(true);
```

Comment:
Sorry, what I meant is that `play` should be fired when this function is called and not when the src is added and it starts playing.